src/main/java/org/folio/consortia/messaging/listener/EventListenerHelper.java

```diff
@@ -32,4 +36,10 @@ protected String getCentralTenantByIdByHeader(MessageHeaders messageHeaders) {
     }
     return null;
   }
+
+  public boolean shouldCreateCentralUserTenantAffiliation(String centralTenantId,
```

Comment:
Suggestion: It could be better to change the name. Currently, the name suggests some creation action for me.

Reply:
For example: isNotCentralTenantAndNotSystemUser()